docs/maintain-guides-society.md

```diff
@@ -0,0 +1,159 @@
+```
+
+A - Accept,  R - Reject,  S - Skeptic
+
+Member | 1 | 2 | 3 | 4 | 5 |
+--- | --- | --- | --- | ---- | --- |
+Vote | A | A | A | R | S |
+Selected | | | X | | | 
+
+In this example, this candidate will be approved to join the society since member 3 was selected as a final outcome. Also, a number of members will be randomly chosen as `skeptics` to vote for the candidates during the rotation period. Since members 5 was chosen as a skeptic, he is required to participate in the voting process. If he does not participate in voting, he will be punished by increase his strike per missing vote. And if you have accumulated too many stikes, your membership will be suspended which means you may need to re-apply the membership and your unclaimed payouts will be slashed. Moreover, each member who voted opposite to the randomly selected vote will be slashed their unclaimed payouts and increase their strikes. In this case, member 4 will be punished.
+
+```bash
+Note:
+- The maximum number of strikes you can have is 10 in Kusama.
+```
+
+These slashed funds will be given to a random member who voted the same as the selected vote as a reward for participating in the vote.
+
+If the candidate wins the vote, they receive their bid reward as a future payout. If the bid was placed by a voucher, they will get the reward that was set during vouching. And the remaining will be given to the candidate.
+
+If the candidate loses the vote, they are suspended and it is up to the suspension judgment origin to determine if the candidate should go through the bidding process again, should be accepted into the membership society, or rejected and their deposit slashed.
+
+
+### 3. Member Phase
+
+Once you become a member of society,  you will get back the deposit that you have reserved during the bidding. A few things you need to be aware of. First, you should vote on candidates who applied for the membership for every rotation period. 
+
+![Society Vote Defender](assets/society/vote_defender.jpg) 
+
+```bash
+defenderVote(approve)
+
+approve - Yes / No
+```
+
+Second, you will need to claim your payout manually by calling `payout()` after a certain period of time.
+
+![Society Payout](assets/society/payout.jpg) 
+
+
+Third, there will have a membership challenge for every `7` days on Kusama. So one of the members will be randomly selected as a defender. Then, other members can vote whether this defender should stay in society. A simple majority wins the vote will determine the outcome of the member. You can take a look at [here](https://polkascan.io/pre/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17) and search for "Existing Members (Challenges)" keyword.
+Besides that, you can earn an extra KSM by helping a user to apply for the membership by requesting a tip. This is useful when a user does not have enough balance to deposit a reserve. And the tip will be given when a user successfully joins the society.
+
+```bash
+Note:
```

Comment:
I recommend against using code blocks for notes everywhere, and focusing on using blockquotes for this.

Reply:
updated